### ISSUE TITLE
Removing QR Code From Shared Records Screen

### ISSUE THREAD

**Issue by philipparkergit** [mycoralhealth/mycoral-patient#22]
Because the use case of wanting to see which records you've shared with your doctors is much more common than the use case of wanting to see which records have been shared with you, I added language in a branch to rephrase it to that more common use case for now (we'll still want the current display as a separate screen later on though so it definitely has value). For that reason we'll need to remove that QR code button from the "Shared Records" screen as it doesn't apply to the new content we're showing.

**Comment by nosequeldeebee** [comment]
Related to #16

**Comment by nosequeldeebee** [comment]
closing this as it should be part of #16
